src/HotChocolate/Stitching/src/Stitching/DependencyInjection/HotChocolateStitchingRequestExecutorExtensions.cs

```diff
@@ -571,6 +573,71 @@ await loadSchema(services, cancellationToken)
                 });
         }
 
+        /// <summary>
+        /// Adds a schema SDL that contains type extensions.
+        /// Extension documents can be used to extend merged types
+        /// or even replace them.
+        /// </summary>
+        /// <param name="builder">
+        /// The <see cref="IRequestExecutorBuilder"/>.
+        /// </param>
+        /// <param name="assembly">
+        /// The assembly from which the type extension file shall be resolved.
+        /// </param>
+        /// <param name="key">
+        /// The resource key of the type extension file
+        /// </param>
+        /// <returns>
+        /// Returns the <see cref="IRequestExecutorBuilder"/>.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="builder"/> is <c>null</c>, or
+        /// <paramref name="assembly"/> is <c>null</c>.
+        /// <paramref name="key"/> is <c>null</c>.
+        /// </exception>
+        public static IRequestExecutorBuilder AddTypeExtensionsFromResource(
+            this IRequestExecutorBuilder builder,
+            Assembly assembly,
```

Comment:
We should evaluate SourceGenerators for Assembly scanning in a later release

Reply:
This has nothing todo with source generators or assembly scanning. It is about resource streams.

Reply:
yup i've seen that later :D